### DISCUSSION TITLE
Point event with 3 possibilities

### DISCUSSION THREAD

**Discussion by habana15** [olivierfriard/BORIS#480] [Q&A]
Hi, 

I've got a simple question about Boris. I need to create an ethogram from scratch and for point event, I need to have 3 possibilities, yes or no (if this event happend or not) but also if this event is not visible.

Thank you for your help ! 

Habana

**Comment by olivierfriard** [comment]
Hi,

you have two possibilities depending on your project:

1) create 3 behaviors (as point events): yes, no and not visible

2) create one behavior with 3 modifiers (yes, no and not visible)

Test these 2 solutions and choose the more adapted

Olivier

**Reply by habana15** [reply]
Hi,

Thank you for your answer. I decided to go with the one behavior with 3 modifiers. 

I have another question : is it possible to extract all the data in one excel file but also with the independant variable? because with synthetic time budget, I can extract all the data I need but without the independant variables.

Thank you for your help.

Habana

**Comment by olivierfriard** [comment]
Hi,
just add 3 behaviors defined as point event: YES, NO and NOT VISIBLE
